processors/filtering/write_annotations.py

```diff
@@ -101,7 +102,7 @@ def process(self):
 
 		# Write to top dataset
 		for label, values in new_data.items():
-			self.add_field_to_parent("annotation_" + label, values, which_parent=self.source_dataset, update_existing=True)
+			self.add_field_to_parent(label, values, which_parent=self.source_dataset, update_existing=True)
```

Comment:
This is indeed an oversight for now, though I would like to have the option for annotation fields to have a 'clean' name; long names are quickly unreadable in spreadsheet software. This can be resolved by initially checking whether an annotation field key already exists in the dataset columns _or_, when annotated datasets are filtered and create a new dataset, if it is not a field registered in the `annotations` table for the parent dataset.

This is a sort-of edge case for now, but I'll try to resolve this next week!